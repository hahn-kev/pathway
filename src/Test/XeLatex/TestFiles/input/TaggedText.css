div.header {
    font-size: 24pt;
    }

.div {
	color: red;
    }

span.header {
    font-size: 18pt;
    }

.span {
	color: blue;
    }