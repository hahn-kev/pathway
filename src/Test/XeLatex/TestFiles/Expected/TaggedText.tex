\documentclass[a4paper]{article} 
\usepackage{float}
\usepackage{grffile}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{fontspec}
\usepackage{fancyhdr}
\usepackage{multicol}
\usepackage{calc}
\usepackage{lettrine}
\usepackage{alphalph}
\usepackage[left=2cm,right=2cm,top=1.99cm,bottom=1.99cm,includeheadfoot]{geometry}
\usepackage{changepage}
\usepackage{needspace}

\parindent=0pt
\parskip=\medskipamount
\begin{document}
\pagestyle{plain}
\sloppy
\setlength{\parfillskip}{0pt plus 1fil}
\font\main="Times New Roman/GR" at 12pt
\font\divmain="Times New Roman/GR":color=ff0000 at 12pt
\font\headermain="Times New Roman/GR" at 18pt
\font\spanmain="Times New Roman/GR":color=0000ff at 12pt

\mbox{} 
\newpage 
\newpage 
\setcounter{page}{1} 
\pagenumbering{arabic} 
\pagestyle{fancy} 

\divmain{Div Tag }

\headermain{Div Header}

\spanmain{ span Tag }\headermain{span Header}

\end{document}
